numba/analysis.py

```diff
@@ -1,2 +1,12 @@
-from numba.core.analysis import * # noqa: F403, F401
-from numba.core.analysis import _use_defs_result # noqa: F401
+from numba.core.errors import (deprecate_moved_module,
+                               deprecate_moved_module_getattr)
```

Comment:
Perhaps wise not to expose these properties in auto-complete:
```suggestion
import numba.core.errors as _errors
```

Reply:
thanks, fixed